package-lock.json

```diff
@@ -111,10 +112,6 @@
       },
       "engines": {
         "node": ">=6.9.0"
-      },
```

Comment:
I'm a little confused at these changes - @ledhed2222 do you know what's causing them?